webpack/scenes/RedHatRepositories/components/EnabledRepositoriesList.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import EnabledRepository from './EnabledRepository';
+
+const EnabledRepositories = ({ repositorySets }) =>
+  repositorySets.map(({ type, repositories }) =>
+    repositories.map(repo => <EnabledRepository key={repo.id} type={type} {...repo} />));
+
+EnabledRepositories.propTypes = {
+  repositorySets: PropTypes.arrayOf(PropTypes.shape({
+    repositories: PropTypes.arrayOf(PropTypes.object).isRequired,
+    type: PropTypes.string.isRequired, // 'kickstart',
```

Comment:
I wonder about these comments.  I can see both their usefulness and their superfluity 😄

Reply:
Yeah I think they could be removed now or in the near future. I left them in there when I copied over the api data to give me some bearings :)